### ISSUE TITLE
knitr::purl()

### ISSUE THREAD

**Issue by carlosvergara** [cjgb/Rmd2R#1]
Check this function to collect R code from an RMarkdown doc.

**Comment by cjgb** [comment]
Yes, I had seen it (after writing my code, I should confess), but I wanted something "ad hoc" for a very particular purpose...
